### DISCUSSION TITLE
Mobile Friendly Hubs

### DISCUSSION THREAD

**Discussion by BraceletMeServices** [mozilla/hubs#5772] [General Questions & Discussion] [answered]
Update : Actually, I now center aligned it and it looks like it opens up a new window, nevermind then. Thanks.(▀̿Ĺ̯▀̿ ̿)
—

Is there a way where my hub can be more mobile friendly? I’m not sure what it looks like on a PC browser yet as my Surface Go had been stolen off of a bus, but if you see here on my website : [www.BraceletMe.Services](http://www.braceletme.services/)

I have it placed on the Home Screen.
On my phone you have to scroll because it won’t fit the page. It’s zoomed in.
——————-
**Questions : 

1. Is there any way to zoom out? 
Or do I make the dimensions smaller ?
-
2.Should I create a new page for it and have it have its own dedicated page ? 

…**

Elliot Goe 
[www.BraceLetMe.Services](http://www.braceletme.services/) 
Single && Looking …
Email : Owner@BraceletMe.Services
Website is under construction.
.

**Comment by matthewbcool** [comment, marked as answer]
If you are running a custom client on hubs-cloud, you can make changes to the UI as you see fit. If you see an issue with hubs.mozilla.com, please create an issue via bug reports. Thank you.

**Comment by BraceletMeServices** [comment]
Thanks.
ExquisiteProducts.Club

Elliot Goe
… SellSome.Online

Hope I hear back from you,
Thanks.

(⁠☞⁠^⁠o⁠^⁠)⁠ ⁠☞Bracelet Me Services
 www.BraceletMe.Services<http://www.braceletme.services/>
  Link to site <⁠(⁠￣⁠︶⁠￣⁠)⁠↗
***@***.***>  (⁠─⁠.⁠─⁠|Email : ***@***.***

Twitter(⁠｡⁠◕⁠‿⁠◕⁠｡⁠)⁠➜ BraceletMeSVS

________________________________
